lib/EpdFont/scripts/reweight-vietnamese-cut.py

```
@@ -0,0 +1,97 @@
+for name, src_name in replacement.items():
+    pen = TTGlyphPen(None)
+    DST.getGlyphSet()[src_name].draw(pen)
+    vglyf[name] = pen.glyph()
+    vhmtx[name] = (DST["hmtx"][src_name][0], DST["hmtx"][src_name][1])
+
+
+def depth(name, seen=()):
+    g = vglyf[name]
+    if not g.isComposite() or name in seen:
+        return 0
+    return 1 + max(depth(c.glyphName, seen + (name,)) for c in g.components)
```

Ruff (0.16.1)

[warning] Consider `(*seen, name)` instead of concatenation

Replace with `(*seen, name)`

(RUF005)